content/graphing/dashboards/timeboard.md

```diff
@@ -46,16 +46,17 @@ https://app.datadoghq.com/event/stream?per_page=30&query=tags:audit%20status:all
 This feature can be enabled by following these simple steps:
 
 1. At the top right corner of a dashboard, click on the gear icon:
-    {{< img src="graphing/dashboards/faq/enable_notifications.png" alt="enable notifications" responsive="true" popup="true">}}
+    {{< img src="graphing/dashboards/faq/enable_notifications.png" alt="enable notifications" responsive="true" popup="true" style="width:30%;">}}
 
 2. Select **Notifications** option and enable the notifications:
-    {{< img src="graphing/dashboards/faq/notifications_pop_up.png" alt=" notifications pop up" responsive="true" popup="true">}}
+    {{< img src="graphing/dashboards/faq/notifications_pop_up.png" alt=" notifications pop up" responsive="true" popup="true" style="width:30%;">}}
 
-## Backup my Timeboard
+## Back up my Timeboard
```

Comment:
again, the term is backup. if you have to separate, at least have the hyphen, but i think backup is better.